packages/backend/src/controllers/discord/templates/tokenRequestMessage.class.ts

```diff
@@ -0,0 +1,11 @@
+import { MessageTemplate } from '../message.class';
+
+export interface TokenRequestMessageOptions {
+  key: string;
+}
+
+export class TokenRequestMessage extends MessageTemplate {
+  readonly template = `:key: **Es wurde ein Zugangstoken angefordert**
+  Zugangstoken lautet: {key}
+  Solltest du den Token nicht angefordert haben - Kein Problem, lösche diese Nachricht einfach`;
```

Comment:
Euer Ehren gegebenenfalls eine Ergänzung eines Satzzeichens. Hochachtungsvoll, Noah Wollenhaupt.
```suggestion
  Solltest du den Token nicht angefordert haben - Kein Problem, lösche diese Nachricht einfach!`;
```